Readme.md

```diff
@@ -1,29 +1,27 @@
 <!-- default badges end -->
 
-# XAF - How to prevent a Detail View from being shown for a List View record
+# XAF - Prevent Showing a Detail View of a List View Object
 
-## Scenario
+This example demonstrates how to prevent a user from invoking a Detail View of an object from a List View.
 
-For certain data forms, a developer needs to limit end-users to editing only through the List View, i.e. without invoking a separate Detail View. This is usually done by activating the inline editing and `MasterDetailMode` = `ListViewAndDetailView` features as described in the following article: [List View Edit Modes](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes).
+To allow users to edit objects directly in a List View, enable the [in-place editing](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes#in-place-editing) and [split layout](https://docs.devexpress.com/eXpressAppFramework/113249/ui-construction/views/list-view-edit-modes#split-layout-the-masterdetailmode-property) functionality.
 
 ## Implementation Details
 
-To accomplish this task, access the [ListViewProcessCurrentObjectController class](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ListViewProcessCurrentObjectController) using the [Frame.GetController<ControllerType>() method](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Frame.GetController--1) and disable the ListViewProcessCurrentObjectController.ProcessCurrentObjectAction.
+Use the [Frame.GetController<ControllerType>](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Frame.GetController--1) method to access the [ListViewProcessCurrentObjectController](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ListViewProcessCurrentObjectController) class and disable the [ListViewProcessCurrentObjectController.ProcessCurrentObjectAction](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ListViewProcessCurrentObjectController.ProcessCurrentObjectAction) property.
 
 ## Additional Information
+  
+This example only covers the case when an XAF application displays a Detail View of an object after a user double-clicks this object in a List View or focuses an object in a List View and presses Enter. To handle other scenarios, extend the controller's code. For example, if you do not want your XAF application to invoke a Detail View after a user clicks the **New** button to create an object in a List View, handle the [NewObjectViewController.ObjectCreating](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.NewObjectViewController.ObjectCreating) event and set its [ObjectCreatingEventArgs.ShowDetailView](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ObjectCreatingEventArgs.ShowDetailView) property to `false`.
```

Comment:
> This example only covers the case when an XAF application displays a Detail View of an object after a user double-clicks this object in a List View or focuses **an** object in a List View and presses Enter.

This is not an issue, but I can't help but explain that this scenario can be interpreted in two ways (and both would be completely correct):

* We started to talk about a specific object and its Detail View in this scenario, therefore, we speak of the same object throughout this sentence.
* We speak everywhere in general about an ability to do something (this is the scenario you explained).

I warn you that this article could be changed during the final editorial review, and there's nothing wrong with it.

> For example, if you do not want your XAF application to invoke a Detail View after a user clicks the **New** button to create an object in a List View, handle the [NewObjectViewController.ObjectCreating](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.NewObjectViewController.ObjectCreating) event and **set its** [ObjectCreatingEventArgs.ShowDetailView](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ObjectCreatingEventArgs.ShowDetailView) property to `false`.

But this looks a technical correctness issue to me because an event can't have a property per se. Therefore, you cannot set the event's property to anything. What do you actually do here, you set the named parameter accessible within the event handler to a value.

We can simplify this, of course, and rather say "set the X property to `false`.

```suggestion
This example only covers the scenario when an XAF application displays a Detail View of an object after a user double-clicks this object in a List View or focuses an object in a List View and presses Enter. To handle other scenarios, extend the controller's code. For example, if you do not want your XAF application to invoke a Detail View after a user clicks the **New** button to create an object in a List View, handle the [NewObjectViewController.ObjectCreating](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.NewObjectViewController.ObjectCreating) event and set the [ObjectCreatingEventArgs.ShowDetailView](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.SystemModule.ObjectCreatingEventArgs.ShowDetailView) property to `false`.
```